hack/format.sh

```diff
@@ -23,6 +23,8 @@ goimports -l -w $@
 # Format import order only after files have been formatted by imports.
 echo "> Format Import Order"
 
+goimports_reviser_opts=${GOIMPORTS_REVISER_OPTIONS:-""}
+
 for p in "$@" ; do
-  goimports-reviser -recursive $p
+  goimports-reviser $goimports_reviser_opts -recursive $p
```

Comment:
How `GOIMPORTS_REVISER_OPTIONS ` will be set on extension side?
The example you have given in PR description will not work,
`${REPO_ROOT}/vendor/github.com/gardener/gardener/hack/format.sh ./cmd ./pkg ./test GOIMPORTS_REVISER_OPTIONS="-imports-order std,project,general,company"`.
Because it takes all the argument on by one and runs 
`goimports-reviser $goimports_reviser_opts -recursive $p` i.e. it will run 

goimports-reviser -recursive GOIMPORTS_REVISER_OPTIONS=-imports-order std,project,general,company, for the example you have given.

Reply:
> Can you explain how GOIMPORTS_REVISER_OPTIONS will be set on extension side?

Very simple, just add something like this in the Makefile of the extension
```
.PHONY: format
format: $(GOIMPORTSREVISER)
	@GOIMPORTS_REVISER_OPTIONS="-imports-order std,project,general,company" \
		$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/format.sh ./cmd ./pkg ./test
```





> Because according to `format.sh` script it takes all the arguments one by one and runs
> 
> ```
> goimports-reviser $goimports_reviser_opts -recursive $p
> ```
> 
> i.e. in your case, it will run
> 
> ```
> goimports-reviser -recursive GOIMPORTS_REVISER_OPTIONS=-imports-order std,project,general,company
> ```
> 
> Which will throw an error.

You are wrong, the command will be `goimports-reviser -imports-order std,project,general,company -recursive <path-to-package>` and I do not see it failing when I am using it. 


> Also is there any use case for why we want this changes

Yes, I want to have the package modules to be in the second import group, not in the third - and this change allows me to control exactly this (well, other flags as well, but I am not interested into them as of now).